specification/loadtestservice/LoadTestService/routes.tsp

```diff
@@ -0,0 +1,692 @@
+  createOrUpdateAppComponents is Azure.Core.Foundations.Operation<
+    {
+      @header 
+      @doc("Content type.")
+      contentType: "application/merge-patch+json",
+
+      @doc("""
+Unique name for the load test, must contain only lower-case alphabetic,
+numeric, underscore or hyphen characters.
+""")
+      @path
+      testId: string;
+
+      @doc("App Component model.")
+      @body
+      body: TestAppComponents;
+    },
+    ResourceCreatedOrOkResponse<TestAppComponents>, {}, ErrorResponseBody
+  >;
+
+  @summary("Get associated app component (collection of azure resources) for the given test.")
+  @doc("Get associated app component (collection of azure resources) for the given test.")
+  @route("/tests/{testId}/app-components")
+  @get
+  @example("./examples/ListTestAppComponents.json", "ListTestAppComponents")
+  getAppComponents is Azure.Core.Foundations.Operation<
+    {
+      @doc("""
+Unique name for the load test, must contain only lower-case alphabetic,
+numeric, underscore or hyphen characters.
+""")
+      @path
+      testId: string;
+    },
+    TestAppComponents, {}, ErrorResponseBody
+  >;
+
+  @summary("Configure server metrics for a test")
+  @doc("Configure server metrics for a test")
+  @route("/tests/{testId}/server-metrics-config")
+  @patch
+  @example("./examples/CreateOrUpdateTestServerMetricsConfig.json", "CreateOrUpdateTestServerMetricsConfig")
+  createOrUpdateServerMetricsConfig is Azure.Core.Foundations.Operation<
+    {
+      @header 
+      @doc("Content type.")
+      contentType: "application/merge-patch+json",
+      
+      @doc("""
+Unique name for the load test, must contain only lower-case alphabetic,
+numeric, underscore or hyphen characters.
+""")
+      @path
+      testId: string;
+
+      @doc("Server metric configuration model.")
+      @body
+      body: TestServerMetricConfig;
+    },
+    ResourceCreatedOrOkResponse<TestServerMetricConfig>, {}, ErrorResponseBody
+  >;
+
+  @summary("List server metrics configuration for the given test.")
+  @doc("List server metrics configuration for the given test.")
+  @route("/tests/{testId}/server-metrics-config")
+  @get
+  @example("./examples/ListTestServerMetricsConfig.json", "ListTestServerMetricsConfig")
+  getServerMetricsConfig is Azure.Core.Foundations.Operation<
+    {
+      @doc("""
+Unique name for the load test, must contain only lower-case alphabetic,
+numeric, underscore or hyphen characters.
+""")
+      @path
+      testId: string;
+    },
+    TestServerMetricConfig, {}, ErrorResponseBody
+  >;
+}
+
+#suppress "@azure-tools/typespec-azure-core/use-standard-operations"
+@tag("TestRun")
+interface LoadTestRun {
+  @summary("Get test run details by name.")
+  @doc("Get test run details by name.")
+  @route("/test-runs/{testRunId}")
+  @get
+  @example("./examples/GetTestRun.json", "GetTestRun")
+  getTestRun is Azure.Core.Foundations.Operation<
```

Comment:
Can the get operations be defined as ResourceRead operations?

Reply:
Updated as suggested.